ansible/roles/satellite-capsule-installation/defaults/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+capsule_repos:
+  - rhel-7-server-rpms
+  - rhel-server-rhscl-7-rpms
+  - rhel-7-server-satellite-maintenance-6-rpms
+  - rhel-7-server-ansible-2.6-rpms
+  - rhel-7-server-satellite-capsule-6.4-rpms
+  - rhel-7-server-satellite-tools-6.4-rpms
+
+firewall_capsule_services:
+  - ssh
+  # - RH-Satellite-6
+# - dns
+# - dhcp
+# - dhcpv6
+# - tftp
+# - libvirt-tls
+# - ldap
+# - ldaps
+# - docker-registry
+# - vnc-server
+
+# List of ports to add into the firewall via Firewalld
+# See README.md for an explanation.
+firewall_capsule_ports:
```

Comment:
small nit: if you don't need all these other ports, maybe cleanup for readability?

Reply:
I have separated comments and used ports. Done